README.md

```diff
@@ -209,14 +204,14 @@ epoch_data.sel(channel=['EEG 001','EEG 002','EEG 003'], sample=range(400))\
         lowpass:           40.0
         highpass:          0.10000000149011612
         lower_limit_rt:    0
-        upper_limit_rt:    5.002
-        reject_threshold:  inf
+        upper_limit_rt:    inf
+        reject_threshold:  None
         n_trials:          200
 
 
 
     
-![png](README_files/README_12_1.png)
+![png](README_files/README_11_1.png)
```

Comment:
Perhaps it is an idea to give the figures a more descriptive name at some point?

Reply:
Those are automatically generated so I wouldn't dare going into that